yktrace/src/tir.rs

```diff
@@ -104,6 +129,13 @@ impl TirTrace {
                             // We have SIR for the callee, so it will appear inlined in the trace
                             // and we only need to emit Enter/Leave statements.
 
+
+                            // If the function has been annotated with do_not_trace, turn it into a
+                            // call.
+                            if callbody.flags & (1 << 2) == (1 << 2) {
```

Comment:
Can we use the symbolic names for the bit fields here?

Reply:
Also, you can just check if the masked result is zero or not.

Reply:
Why do I need check that?

Reply:
You don't have to, it's just more conventional and (maybe) slightly faster to check for non-zero:

```
if callbody.flags & bodyflags::DO_NOT_TRACE != 0 {
```

Reply:
I misread your comment.